install/tools/istio_vm_common.sh

```diff
@@ -189,18 +189,19 @@ function istioBootstrapVM() {
 
   istio_provision_certs $SA
 
-  local ISTIO_FILES=${ISTIO_FILES:-.}
+  local ISTIO_IO=${ISTIO_BASE:-${GOPATH:-$HOME/go}}/src/istio.io
+  local ISTIO_STAGING=${ISTIO_STAGING:-$ISTIO_IO/istio/bazel-bin/vm}
```

Comment:
This directory (and all of the other bazel-* dirs) is actually a symlink to `$HOME/.cache/bazel/...` - we should be able to create directories there, but they're ephemeral.

Reply:
my understanding is the script will be packaged in our istio release binary, correct?  If so, is bazel stuff mandatory?  I'd hope not, if so, i would prefer a default dir not mention bazel to avoid confusion.

Reply:
AFAIK we can create files and dirs - this can go away once we have all .deb files available
for download.

Reply:
Resolved - will just use the structure that downloadIstio creates ( and updated the script to get the files we need ).

Long term the deb files will be in an apt repo, and we can include the helpers for VM in the debian ( a bit too late for Monday, will try to have a PR ready just in case )